poucave/html/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Delivery System Status</title>
+  <title></title>
```

Comment:
Does it make sense to have a placeholder, like "Loading configuration..." in case something goes wrong with the configuration request?